Net07.HW/Net07.WorkWithDate/Net07.WorkWithDate/Program.cs

```diff
@@ -0,0 +1,160 @@
+﻿using System;
+
+namespace Net07.WorkWithDate
+{
+    class Program
+    {
+        enum DaysInMonths
+        {
+            jan = 31,
+            lFeb = 29,
+            Feb = 28,
+            mar = 31,
+            apr = 30,
+            may = 31,
+            jun = 30,
+            jul = 31,
+            aug = 31,
+            sep = 30,
+            oct = 31,
+            nov = 30,
+            dec = 31
+        }
+        enum DaysOfWeek
+        {
+            Monday,
+            Tuesday,
+            Wednsday,
+            Thursday,
+            Friday,
+            Saturday,
+            Sunday
+        }
+        static void Main(string[] args)
+        {
+            GetDayOfWeekEasyLevel();
+            GetDayOfWeekMiddleLevel();
+            GetDayOfWeekHardLevel();
+        }
+
+        private static void GetDayOfWeekHardLevel()
+        {
+            Console.WriteLine("Введите дату ");
+            string dateStr = Console.ReadLine();
+            string[] strArr = dateStr.Split('.');
+            int dayInDate = int.Parse(strArr[0]), monthInDate = int.Parse(strArr[1]), yearInDate = int.Parse(strArr[2]), leapDays = CalcLeapDays(yearInDate);
+            int totalDays = 365 * (yearInDate - 1) + leapDays;
+            for (int i = 1; i < monthInDate; i++)
```

Comment:
Please, use `{`, `}`. If you add one more statement, you may forget to add brackets. This mistake seems to be stupid, but it's too frequent to don't care about it.

Reply:
fixed